portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -96,13 +100,22 @@ private int convertToInt(String beingconverted) {
       convertee = Integer.parseInt(beingconverted);
     } catch (NumberFormatException e) {
       System.err.println("Could not convert to int: " + beingconverted);
-      return 5;
+      return 1;
```

Comment:
You don't need to change this since we're moving on to the starter project, but instead of returning this function show throw an Exception for error cases instead - otherwise, callers will not be  able to figure out the difference between a valid "1" and an error "1".